styles.css

```diff
@@ -144,8 +144,9 @@ html {
   color: #EFB7EC;
   display: block;
   align-self: flex-start;
-  padding-top: 70px;
+  margin-top: 70px;
```

Comment:
Ah ha! Another example of how things are affected by adjustments to the box-model. Thank you, Scott!